getting-started/templates/systemlink-values.yaml

```diff
@@ -496,10 +496,22 @@ dataframeservice:
   ## Ingress configuration
   ##
   ingress:
-    ## Override the default request size limit for an nginx ingress controller.
+    ## Increase the maximum HTTP request body size from the nginx default. Should be
+    ## set to the same size as requestBodySizeLimitMegabytes. For more information,
+    ## see requestBodySizeLimitMegabytes.
     ##
     annotations:
-      nginx.ingress.kubernetes.io/proxy-body-size: 256m
+      nginx.ingress.kubernetes.io/proxy-body-size: 250m
```

Comment:
This assumes an nginx ingress controller. If the customer is using something else then this won't do anything. We only officially support nginx for on-prem at the moment, but it would be good to at least document this restriction.

Reply:
Added a note that this only applies if an nginx ingress controller is used.